climate.py

```diff
@@ -94,6 +94,7 @@ def __init__(self, client, device, fahrenheit=True):
         self._client = client
         self._device = device
         self._fahrenheit = fahrenheit
+        self._attrs = {'power':0}
```

Comment:
Maybe this should just be `{}` initially, to indicate that no power is available, rather than misleadingly reporting "zero watts"?